indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -304,82 +290,48 @@ public TaskStatus call()
-                        command.add(StringUtils.format("-Ddruid.tlsPort=%d", tlsChildPort));
+
+                        command.addSystemProperty("druid.host", childHost);
+                        command.addSystemProperty("druid.plaintextPort", childPort);
+                        command.addSystemProperty("druid.tlsPort", tlsChildPort);
 
                         // Let tasks know where they are running on.
                         // This information is used in native parallel indexing with shuffle.
-                        command.add(StringUtils.format("-Ddruid.task.executor.service=%s", node.getServiceName()));
-                        command.add(StringUtils.format("-Ddruid.task.executor.host=%s", node.getHost()));
-                        command.add(
-                            StringUtils.format("-Ddruid.task.executor.plaintextPort=%d", node.getPlaintextPort())
-                        );
-                        command.add(
-                            StringUtils.format(
-                                "-Ddruid.task.executor.enablePlaintextPort=%s",
-                                node.isEnablePlaintextPort()
-                            )
-                        );
-                        command.add(StringUtils.format("-Ddruid.task.executor.tlsPort=%d", node.getTlsPort()));
-                        command.add(
-                            StringUtils.format("-Ddruid.task.executor.enableTlsPort=%s", node.isEnableTlsPort())
-                        );
-                        command.add(StringUtils.format("-Dlog4j2.configurationFactory=%s", ConsoleLoggingEnforcementConfigurationFactory.class.getName()));
+                        command.addSystemProperty("druid.task.executor.service", node.getServiceName());
+                        command.addSystemProperty("druid.task.executor.host", node.getHost());
+                        command.addSystemProperty("druid.task.executor.plaintextPort", node.getPlaintextPort());
+                        command.addSystemProperty("druid.task.executor.enablePlaintextPort", node.isEnablePlaintextPort());
+                        command.addSystemProperty("druid.task.executor.tlsPort", node.getTlsPort());
+                        command.addSystemProperty("druid.task.executor.enableTlsPort", node.isEnableTlsPort());
+                        command.addSystemProperty("log4j2.configurationFactory", ConsoleLoggingEnforcementConfigurationFactory.class.getName());
 
-                        // These are not enabled per default to allow the user to either set or not set them
-                        // Users are highly suggested to be set in druid.indexer.runner.javaOpts
-                        // See org.apache.druid.concurrent.TaskThreadPriority#getThreadPriorityFromTaskPriority(int)
-                        // for more information
-                        // command.add("-XX:+UseThreadPriorities");
-                        // command.add("-XX:ThreadPriorityPolicy=42");
```

Comment:
are these not required anymore?

Reply:
They were commented out.  commented out code is pointless, it just generates questions of "is this necessary" and then sits there from release to release because nobody knows why it's there or when it can be uncommented.  I cleaned it up.  If it's necessary, they can be added and not commented out.